frontend/src/__tests__/UnorderedList.test.js

```diff
@@ -33,17 +28,5 @@ describe('Unordered List', () => {
       </ThemeProvider>,
       div,
     )
-    ReactDOM.render(
```

Comment:
Might be worth testing one with a `listStyleType` passed in

Reply:
👍